modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -4,19 +4,27 @@
 import * as FormData from "form-data";
 {{/node}}
 {{/platforms}}
+{{#platforms}}
+{{^deno}}
 // typings of url-parse are incorrect...
 // @ts-ignore 
 import * as URLParse from "url-parse";
-import { Observable, from } from {{#useRxJS}}'rxjs'{{/useRxJS}}{{^useRxJS}}'../rxjsStub'{{/useRxJS}};
+{{/deno}}
+{{/platforms}}
+import { Observable, from } from {{#useRxJS}}'rxjs'{{/useRxJS}}{{^useRxJS}}'../rxjsStub{{#platforms}}{{#deno}}.ts{{/deno}}{{/platforms}}'{{/useRxJS}};
 
+{{#platforms}}
+{{^deno}}
 {{#frameworks}}
 {{#fetch-api}}
-export * from './isomorphic-fetch';
+export * from './isomorphic-fetch{{#platforms}}{{#deno}}.ts{{/deno}}{{/platforms}}';
```

Comment:
this change looks redundant, since this whole line is not generated for Deno (which uses a built-in `fetch` anyway)

Reply:
Thanks! I fixed.

Reply:
I have checked that if we can use Inversify on Deno.

## Deno Third Party Modules
Inversify is not listed in [Deno Third Party Modules](https://deno.land/x).

## Pika
https://www.pika.dev/search?q=inversify
message: "Package found! However, no web-optimized "module" entrypoint was found in its package.json manifest."

## jspm

```
$ cat inversify_sample.ts

import "https://jspm.dev/reflect-metadata";

// @deno-types="https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/inversify.d.ts"
import { Container, injectable } from "https://jspm.dev/inversify@5.0.1";

var container = new Container();

$ ~/data/work$ deno run --reload -c tsconfig.json inversify_sample.ts
Download https://jspm.dev/reflect-metadata
Download https://jspm.dev/inversify@5.0.1
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/inversify.d.ts
Download https://jspm.dev/npm:inversify@5.0.1/_/2b7febed.js
Download https://jspm.dev/npm:inversify@5.0.1/_/2e4eea2e.js
Download https://jspm.dev/npm:inversify@5.0.1/_/d819b7b6.js
Download https://jspm.dev/npm:inversify@5.0.1/_/8b88d942.js
Download https://jspm.dev/npm:inversify@5.0.1!cjs
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/constants/metadata_keys
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/container/container
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/constants/literal_types
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/container/container_module
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/injectable
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/tagged
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/named
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/inject
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/optional
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/unmanaged
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/multi_inject
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/target_name
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/post_construct
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/planning/metadata_reader
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/utils/id
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/interfaces/interfaces
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/annotation/decorator_utils
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/syntax/constraint_helpers
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/utils/serialization
Download https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/utils/binding_utils
Download https://jspm.dev/npm:@jspm/core@2/nodelibs/process
Download https://jspm.dev/npm:reflect-metadata@0.1.13!cjs
Download https://jspm.dev/npm:@jspm/core@2.0.0-beta.7/_/de9a8dfb.js
error: Import 'https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/utils/binding_utils' failed: 404 Not Found
Imported from "https://cdn.jsdelivr.net/npm/inversify@5.0.1/dts/inversify.d.ts:21"

$ echo $?
1
```

Inversify may work on Deno via jspm. But I couldn't load the Type Definition file.
Inversify may officially support Deno in the future.
At the moment, I think it's better not to support by openapi-generator yet.
This time, I added a description to the documentation that it is not currently supported.